api/app/src/main/kotlin/packit/service/RoleService.kt

```diff
@@ -60,7 +60,7 @@ class BaseRoleService(
 
     override fun deleteRole(roleName: String)
     {
-        if (!roleRepository.existsByName(roleName))
+        if (!roleRepository.existsByName(roleName) || roleName == "ADMIN")
```

Comment:
Should probably be a different error if they've tried to delete ADMIN

Reply:
...And should it also be an error if an attempt is made to delete a username role instead of deleting the user? (shouldn't be available in the FE, but might happen via the API... )

Reply:
true admin one different error... 2nd comment that is done already as part of user-role update